index.js

```diff
@@ -125,11 +125,10 @@ function configurePassport(client, type) {
       if (process.env.VERBOSE === 'true') {
         console.log('access_token', tokenset.access_token);
         console.log('id_token', payload);
-      } else {
-        console.log('user.name', user.email);
-        console.log('user.icn', user.fediamMVIICN);
-        console.log('access_token digest', new shajs.sha256().update(user.access_token).digest('hex'));
       }
+      console.log('user.name', user.email);
+      console.log('user.icn', user.fediamMVIICN);
+      console.log('access_token digest', new shajs.sha256().update(user.access_token).digest('hex'));
```

Comment:
If we're displaying this in non-verbose mode, we may as well show in verbose mode too.